### PR TITLE
Editorial changes for the CF-1.12 release documents

### PR DESCRIPTION
**Pull request by davidhassell**
No associated issue.

# Release checklist
- [x] Authors updated in `cf-conventions.adoc`? Add in two places: on line 3 and under `.Additional Authors` in `About the authors`.
- [x] Next version in `cf-conventions.adoc` up to date? Versioning inspired by [SemVer](https://semver.org).
- [x] `history.adoc` up to date?
- [x] Conformance document up to date?

# For maintainers
After the merge remember to delete the source branch.
Tags are set at the conclusion of the annual meeting; until then, `main` always is a draft for the next version.

### CODE REVIEW

**Review by cofinoa — APPROVED** [review]
I have updated the .zenodo.json

Checked CITATION.cff

Checked DOI values and links to DOI in PDF and HTML artifacts

File: .zenodo.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
...d with the netCDF Application Programmer Interface. The conventions define metadata that provide a definitive description of what the data in each variable represents, and of the spatial and temporal properties of the data. This enables users of data from different sources to decide which quantities are comparable, and facilitates building applications with powerful extraction, regridding, and display capabilities.</p><p>The CF conventions generalize and extend the COARDS conventions. The extensions include metadata that provides a precise definition of each variable via specification of a standard name, describes the vertical locations corresponding to dimensionless vertical coordinate values, and provides the spatial coordinates of non-rectilinear gridded data. Since climate and forecast data are often not simply representative of points in space/time, other extensions provide for the description of coordinate intervals, multidimensional cells and climatological time coordinates, and indicate how a data value is representative of an interval or cell. This standard also relaxes the COARDS constraints on dimension order and specifies methods for reducing the size of datasets.</p>",
```
Comment (JonathanGregory) [inline .zenodo.json:4]: What's the reason for this change? Is it OK to have <p> in the Zenodo description?

**Review by JonathanGregory — APPROVED** [review]

File: .zenodo.json (hunk 1 above)
Comment (cofinoa) [reply .zenodo.json:4]: Yes, it's ok for Zenodo. It's needed if we want to split both paragraphs i. the abstract. All previous versions, include them, and also I have removed the [COARDS] and [NetCDF] text used as links.

**Review by cofinoa — APPROVED** [review]

File: version.adoc [hunk 2]
```diff
@@ -8,16 +8,18 @@ or with the commandline switch `-a final`.
...
 // :final:
 
-// No changes needed beyond this line
 ifdef::final[]
+// Change the DOI prior to releasing the next version
+:doi: 10.5281/zenodo.14275599
+// No changes needed beyond this line
 :current-version: {version}
 :current-version-as-attribute: {version}
-:doi: 10.5072/zenodo.FFFFFF
```
Comment (cofinoa) [inline version.adoc:15]: don't remove it, ad here the actual pre-resrved DOI for v1.12. 

We will need to handle this better in the future

**Review by cofinoa — APPROVED** [review]

File: version.adoc (hunk 2 above)
Comment (davidhassell) [reply version.adoc:15]: Agreed. I moved the `:doi:` definition a few lines up, so it's still there.
